azure/terraform/dbfirewallrules.rego

```diff
@@ -46,11 +46,11 @@ db_firewall = false {
     azure_attribute_absence["db_firewall"]
 }
 
-db_firewall_err = "SQL Server Firewall rules allow access to any Azure internal resources" {
+db_firewall_err = "SQL Server Firewall rules configuration currently allowing everyone get full access" {
```

Comment:
`SQL Server Firewall rule configuration allowing full inbound access to everyone`

Reply:
updated.